metrics/time/README.md

```diff
@@ -0,0 +1,10 @@
+# Time metrics tests
+
+This section includes metrics tests focused on measure the time interval[s] of a determined
+number of containers executing a defined workload. This kind of metrics tests could provide
+a general overview about the time spent by a flow, however the use of these should be
+complemented with more tools in order to find the hot spot.
+
+- `docker_workload_time.sh`: measures the complete workload of a container using docker. The
+   workload of this test is to execute a `true`, the reason behind this is not to add overhead
+   and measure the container flow execution.
```

Comment:
This sentence is confusing. Here's my take:
measures the time taken for a container using Docker\* to complete a workload. In this test, the workload is to execute a `true`. By using this workload, the test does not add overhead when measuring the container flow execution time.